### DISCUSSION TITLE
How to bulk import a CSV with an Object Custom Field?

### DISCUSSION THREAD

**Discussion by thiagomarafeli** [netbox-community/netbox#15030] [Q&A] [answered]
I'm trying to bulk import a CSV file and one of the columns is my cf_instance_type custom field, which is an Object (in this case, a device type)

When I try to import it in VM, where my custom field is, it raises an error saying `Record 1 cf_instance_type: Object not found t3_micro`

I have tried using the slug instead of the model, but it raises the same error

I found this problem on issues #13803 and #5728 , but I couldn't understand what is the solution

Any body could explain to me how do I import a CSV with a custom field of type "Object" using Bulk Import?

**Comment by candlerb** [comment, marked as answer]
I would guess (but have not tested) that you have to put the numeric ID of the object you're referring to.
